packages/fx-core/src/plugins/resource/bot/v2/plugin.ts

```diff
@@ -163,10 +173,15 @@ export class TeamsBotV2Impl {
       workingPath
     );
 
+    // For backward compatibility, get resource id from both key `botWebAppResourceId` and `resourceId`
```

Comment:
Good to see the handle of backward compatibility, I am curious on the details and we can sync more offline.